app/views/reports/elements/_step_element.html.erb

```diff
@@ -27,9 +27,10 @@
     <div class="row">
       <div class="col-xs-12 ql-editor">
         <% if strip_tags(step.description).present? %>
-          <%= auto_link(step.description,
-                        link: :urls,
-                        html: { target: '_blank' }).html_safe %>
+          <%= auto_link(smart_annotation_parser(step.description),
```

Comment:
This should be done globally, if at all, and I suggest we add it as a separate task (it's not specifically related to this issue, thus this PR).